### PR TITLE
Fix missing gem "multipart-post" for TestFairy provider

### PR DESCRIPTION
**Pull request by giltsl**

### CODE REVIEW

File: lib/dpl/provider/testfairy.rb [hunk 1]
```diff
@@ -70,10 +72,14 @@ def signing_apk(instrumentedFile)
...
       def download_from_url(url)
-        puts "downloading  from #{url} "
+        puts "downloading from #{url} "
+        url = "#{url}?api_key=#{option(:api_key)}"
         uri = URI.parse(url)
         instrumentedFile = Net::HTTP.start(uri.host, uri.port) do |http|
-          resp = http.get(uri.path)
+          resp = http.get "#{uri.path}?#{uri.query}"
+          if resp.code == "302"
+            resp = Net::HTTP.get_response(URI.parse(resp.header['location']))
```
Comment (flowlo) [inline lib/dpl/provider/testfairy.rb]: [`Net::HTTP` docs on redirection, FYI](http://ruby-doc.org/stdlib-2.2.3/libdoc/net/http/rdoc/Net/HTTP.html#class-Net::HTTP-label-Following+Redirection). You will probably only follow one redirect, which may make sense, or not.

**Comment by flowlo** [comment]
I suggest squashing all those `api_key added *` commits.

File: lib/dpl/provider/testfairy.rb (hunk 1 above)
Comment (gmegidish) [inline lib/dpl/provider/testfairy.rb]: Yes. We will only redirect once, it makes complete sense.

**Comment by gmegidish** [comment]
@flowlo squashed! :)

**Pull request merged**